react/ProductList.tsx

```diff
@@ -54,7 +54,7 @@ const ProductList: StorefrontFunctionComponent<Props> = ({
     ))
 
   return (
-    <Fragment>
+    <div>
```

Comment:
Add an `id` to this `div` to avoid this kind of mistake in the future, maybe `unavailable-items-wrapper`.

Reply:
It's not clear to me how adding an `id` here would avoid this problem in the future. I think a comment is more effective in this case.

Reply:
I gave this some thought and I think it is better to leave it out as it is. Adding a comment would just pollute the code and adding an `id` would just make the reader wonder where that `id` is used (i.e. we would just add a new layer to the same problem). I'll commit myself to properly test these kinds of PRs in the future to prevent this issue from happening again.

Reply:
I thought that adding an `id` would make the reader understand that this `div` is used, it is necessary, which would reduce the burden of remembering every time.
What about when you go on vacation? Who will be this `div` guardian? We need this explicit somewhere (maybe a comment, I don't know).

Reply:
If we add an `id`, an attentive reader would think "but wait, this `id` is not used anywhere, we can safely remove it". They would open a PR, test it, verify everything still works properly, we would approve (because our memory sucks), and everyone would be happy. Except that we would return exactly to the same point we are right now.

The same ~would~ should not happen with this `div`. Removing it breaks the interface. We don't need a guardian for this `div` nor remember that this `div` is important as long as people test their changes.

My concern about adding comments is that they become outdated really fast. One day we will change the blocks structure of this app and the `div` will actually become pointless, yet we will not remove it from the code because of the comment we added today.

Let's do this: I'll add a comment along with a link to this PR. In the future, if we're unsure about the necessity of this `div`, we can just check this discussion and decide.

Reply:
Great! Approved ✅